src/components/LModelCard.vue

```diff
@@ -0,0 +1,31 @@
+<script lang="ts" setup>
+export type LModelCardProps = {
+	name: string;
+	subtitle: string;
+	description: string;
+	type: string;
+};
+defineProps<LModelCardProps>();
+</script>
+
+<template>
+	<div
+		class="flex flex-col gap-y-3 rounded-[12px] border-[0.5px] border-majorelle-300 bg-neutral-white px-6 py-9 shadow-[0_20px_20px_0_rgba(100,77,249,.1)] max-lg:h-[250px] max-lg:w-[300px] lg:h-[200px] lg:w-[400px]"
```

Comment:
> shadow-[0_20px_20px_0_rgba(100,77,249,.1)]
my eyes are burning, please add it in the `<style>` below it will be easier to read. Inline pure css in great with simple values with tailwind, but this is unreadable imo